app/org/sagebionetworks/bridge/models/accounts/Withdrawal.java

```diff
@@ -0,0 +1,35 @@
+package org.sagebionetworks.bridge.models.accounts;
+
+import org.joda.time.DateTime;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Request to withdraw from the current study.
+ *
+ */
+public final class Withdrawal {
```

Comment:
Also, I feel like it's worth unit testing JSON deserialization, just as a sanity test.